src/main/kotlin/lotto/domain/selling/ExchangeResult.kt

```diff
@@ -0,0 +1,16 @@
+package lotto.domain.selling
+
+data class ExchangeResult(val details: Map<Int, Int>) {
+    private val totalPrizeMoney = details.toSortedMap().entries.sumBy { findPrizeMoney(it.key) }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private val totalPrizeMoney: Int = details.keys.sumBy(::findPrizeMoney)
```